doc/api/http.md

```diff
@@ -917,6 +917,66 @@ Example:
 var contentType = response.getHeader('content-type');
 ```
 
+### response.getHeaderNames()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Array}
+
+Returns an array containing the names of the current outgoing headers. Do not
+expect the names to be sorted in any particular manner. All header names are
+lowercase.
```

Comment:
I've gone ahead and just removed the sentence. We don't make any mention of sorted-ness of arrays for any other APIs that I could find that return arrays (e.g. `eventEmitter.eventNames()`).